packages/host/package.json

```diff
@@ -18,8 +18,8 @@
     "lint:fix": "concurrently \"pnpm:lint:*:fix\" --names \"fix:\"",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
+    "lint:js": "eslint . --quiet --cache",
```

Comment:
There was a deprecation warning that we can ignore but I didn't find a way to turn it off from `eslintrc.js` so I chose to use `--quiet` options to turn off `warning` message.